testutil/integration/app.go

```diff
@@ -141,15 +165,6 @@ func NewIntegrationApp(
 	_, err = bApp.Commit()
 	require.NoError(t, err, "failed to commit")
 
-	// TODO_HACK(@Olshansk): I needed to set the height to 2 so downstream logic
-	// works. I'm not 100% sure why, but believe it's a result of genesis and the
-	// first block being special and iterated over during the setup process.
-	cometHeader := cmtproto.Header{
-		ChainID: appName,
-		Height:  2,
-	}
-	ctx := sdkCtx.WithBlockHeader(cometHeader).WithIsCheckTx(true)
-
 	return &App{
```

Comment:
The `sdkCtx: ctx,` line is out of reach of the in-line comment window but this would need to be updated to:

```go
	sdkCtx: &sdkCtx,
```